packages/storybook-react/stories/extension-link/linkify.tsx

```diff
@@ -0,0 +1,30 @@
+import * as linkify from 'linkifyjs';
+import React from 'react';
+import { LinkExtension } from 'remirror/extensions';
+import { Remirror, ThemeProvider, useRemirror } from '@remirror/react';
+
+const findAutoLinks = (str: string) =>
+  linkify.find(str).map((link) => ({
+    text: link.value,
+    href: link.href,
+    start: link.start,
+    end: link.end,
+  }));
+
+const isValidUrl = (input: string) => linkify.test(input);
+
+const Linkify = (): JSX.Element => {
```

Comment:
Sorry to be pedantic, but could this be called `LinkifyJS` instead?

Reply:
Absolutely, let me update this.